lib/bap/bap_inject_externals.mli

```diff
@@ -0,0 +1,9 @@
+open Bap_types.Std
+open Bap_image_std
+open Bap_sema.Std
+open Bap_disasm_std
+
+(** [run prg insn spec] performs search for relocations in [spec] and
```

Comment:
performs search = searches. Keep away from the bureaucratic style

Also, try to document functions, not in terms of how they are implemented, but in terms of what they do, i.e., in a more declarative style. Also, try to minimize the context that is required to understand what the function is doing.

The documentation shall answer the following questions:

1) what is sought in the spec exactly (specify the attributes)
2) what is inserted into the project (explain how functions are named, note that they are tagged with the [synthetic] attribute)
3) what are the assumptions, if any.